cmd/kubeadm/app/cmd/reset.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/spf13/cobra"
 
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
-	"k8s.io/kubernetes/cmd/kubeadm/app/phases/kubeconfig"
+	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
```

Comment:
Similar to my last comment. I really dislike how many package alias we use. Why can't we just use constants here?  For me, what that shows, is that our package names are not well thought out.